### DISCUSSION TITLE
ChatGPT-powered nextra: Get exact answers to your questions

### DISCUSSION THREAD

**Discussion by benjaminshafii** [shuding/nextra#1546] [Show and tell]
![image](https://user-images.githubusercontent.com/11430621/220747631-f69cf532-e464-4ec1-ac04-75018f77d561.gif)

Hey just wanted to share that we built a gpt-3 powered nextra.

## How it works

* during build we index .mdx files with OpenAI Embeddings 
* when you search, we use semantic search with ) to find the most relevant snippets
* we then ask GPT-3 to give a summary of the snippets

Link:
https://github.com/another-ai/chat-gpt-powered-nextra

**Comment by skovy** [comment]
This is awesome! I've been working on a simple productized version of this for people that want something like this without having to spin up the infra themselves. Would love to hear your and other's feedback: https://asktro.com
